pkg/authorization/api/v1/register.go

```diff
@@ -48,6 +56,52 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 
 		&RoleBindingRestriction{},
 		&RoleBindingRestrictionList{},
+	}
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		append(types,
+			&unversioned.Status{}, // TODO: revisit in 1.6 when Status is actually registered as unversioned
```

Comment:
are we planning to do this before or after the 1.6 rebase? seems like we'll have to rework this for the metav1 stuff